utam-compiler/src/main/java/utam/compiler/grammar/UtamElement.java

```diff
@@ -88,7 +88,7 @@ public UtamElement(String name, UtamSelector selector) {
 
   @JsonCreator
   UtamElement(
-      @JsonProperty(value = "type") String type, // optional for actionable
+      @JsonProperty(value = "type") String[] type, // optional for actionable
```

Comment:
Probably right for now. However, a forthcoming update will need to change this type to `JsonNode`, and do the proper evaluation of the content to disallow invalid values.

Reply:
Huh. Turns out `defaultValue = "[]"` doesn't actually provide a default value for a missing member in the JSON. All it does is provide documentation of what the default value _would be_ if you're generating a schema or something. So I'm adding the default value here, but it will have no effect on the actual code in place.